backend/src/services/preferences/wishlist.service.ts

```diff
@@ -0,0 +1,43 @@
+import { prisma } from '@data/prisma-client';
+
+import type { WishlistResponseDto } from '@autoline/shared';
+import type { WishlistInput } from '@common/types/types';
+
+const setWishlist = async (
+  input: WishlistInput,
+): Promise<WishlistResponseDto> => {
+  const { user_id, model_id, complectation_id } = input;
+
+  const wishlist = await prisma.user_Wishlist.findFirst({
+    where: {
+      user_id,
+      model_id,
+      complectation_id,
+    },
+  });
+
+  const { id: newWishlistId } = wishlist
+    ? await prisma.user_Wishlist.delete({
+        where: {
+          id: wishlist.id,
+        },
+      })
+    : await prisma.user_Wishlist.create({
```

Comment:
Didn't Prisma generate the `User_Wishlist` with upper _U_ at the start?

Reply:
@oleksandr-danylchenko I can't find out how to fix this. Prisma generates model with first letter in lower case `user_Wishlist`. Other users faced this issue too https://github.com/prisma/prisma/issues/5035

Reply:
Ohhh... I got this. Prisma, by default, uses the `camelCase` for the tables. So it just uses the lowercase for all the generated tables - https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/use-custom-model-and-field-names#:~:text=Prisma%20has%20a%20naming%20convention%20of%20camelCasing